kernel/src/process.rs

```diff
@@ -185,7 +185,8 @@ impl<'a> Process<'a> {
 
     pub fn setup_mpu(&self, mpu: &::platform::MPU) {
         let data_start = self.memory.as_ptr() as usize;
-        let data_len = 12;
+        let data_len = closest_power_of_two(self.memory.len() as u32)
```

Comment:
If you have a 14 kB process, this should make a 16 kB MPU region, right?

Does the kernel ensure that no other process starts in the remaining 2 kB of memory?